server_admin/topics/authentication/x509.adoc

```diff
@@ -289,6 +289,14 @@ If {project_name} is behind a reverse proxy, you generally need to configure the
 
 You use this provider when your {project_name} server is behind an HAProxy reverse proxy. Use the following configuration for your server:
 
+ifeval::["{kc_dist}" == "quarkus"]
+[source,bash]
+----
+kc.[sh|bat] build --spi-x509cert-lookup-provider=haproxy
+kc.[sh|bat] start --spi-x509cert-lookup-haproxy-ssl-client-cert=SSL_CLIENT_CERT --spi-x509cert-lookup-haproxy-ssl-cert-chain-prefix=CERT_CHAIN --spi-x509cert-lookup-haproxy-certificate-chain-length=10
+----
+endif::[]
+ifeval::["{kc_dist}" == "wildfly"]
```

Comment:
As this is about configuring the server it should be documented in https://www.keycloak.org/server/reverseproxy, and not here.